Microsoft.Toolkit.Uwp.UI.Controls/ColumnResizer/ColumnResizer.Events.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// ColumnResizer is a UI control that add the resizing functionality to a Grid Column.
+    /// </summary>
+    public partial class ColumnResizer
```

Comment:
Wouldn't it be better to match WPF and call it a GridSplitter? Also we should allow resizing rows and not just columns